docs/javascript/guidelines.rst

```diff
@@ -5,6 +5,7 @@ Guidelines
 * Use camelCase for variables and **not** underscores or dashes
 * Use **dot** annotation ``.`` for javascript file naming
 * Use **single-quotes** ``'`` for **all** values
+* Use a maximum length of 120 characters for JS code
```

Comment:
I think that other than the maximum length of 120 we should also have a recommended length of 80 columns. Developers are generally expected to keep to 80 columns but are permitted to use up to 120 when using less makes the code less readable.